src/PandoraMonitoring.cc

```diff
@@ -991,6 +996,49 @@ void PandoraMonitoring::ViewEvent()
 
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+void PandoraMonitoring::SaveAndViewEvent(const std::string savePath)
+{
+    // TODO: We should be checking the path exists/check SavePic is capable of
+    // dealing with the case it isn't.
+
+    this->InitializeEve();
+    // TODO: For whatever reason, the first save has a different camera angle in the 3D view.
+    // This is then sorted for the rest of the events.
```

Comment:
I've resolved this by resetting the camera for each view before saving. Calling `SetEveDisplayParameters` didn't seem to help since it was already being called over in `VisualMonitoring` in LArContent.